securedrop/tests/test_i18n_tool.py

```diff
@@ -347,7 +352,11 @@ def r():
         ])
         assert 'l10n: updated nl' in r()
         assert 'l10n: updated de_DE' not in r()
-        message = str(git('--no-pager', '-C', 'securedrop', 'show',
-                      _cwd=d))
+        if six.PY2:
```

Comment:
Likewise here for `six.string_type(git(...))`

Reply:
`str.u` does not work for our test case and creates exception on `six` itself.